lms/services/lti_outcomes.py

```diff
@@ -82,12 +81,6 @@
 class LTIOutcomesRequestParams(NamedTuple):
     """Common parameters used by all LTI Outcomes Management requests."""
 
-    consumer_key: str
-    """OAuth 1.0 consumer key used to sign the request."""
-
-    shared_secret: str
-    """OAuth 1.0 shared secret used to sign the request."""
```

Comment:
These aren't really params for the LTI outcomes request, but rather things that are needed in order to sign the request, so removing them from here makes sense :+1: